index.html

```diff
@@ -12,26 +12,35 @@
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
     <header class="header">
         <nav class="navbar">
             <label class="mylogo">RASHID</label>
             <i class="menu-icon fa-solid fa-bars"></i>
+            <ul class="menu-list">
+                <li><a href="">Portfolio</a></li>
+                <li><a href="">About</a></li>
+                <li><a href="">Contact</a></li>
+            </ul>
         </nav>
         <section class="section">
-            <h1 class="heading">I’m Rashid
```

Comment:
- [OPTIONAL] Kindly work on the font-weight and size of this section so it looks like Figma's design, and fix the spacing between the navigation bar as well. sorry if the previous reviewer did not mention this.
![Captureheader](https://user-images.githubusercontent.com/81444338/187610236-8f587c7a-92db-46ff-9040-b8dea8a6168e.PNG)